src/main.py

```diff
@@ -1,2 +1,2 @@
 # python3.7.5
-print('hello world yashio!')
+print(f'hello world yashio!')
```

Comment:
**[reviewdog]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
F541 f-string is missing placeholders